_posts/2020-11-08-organizing-your-folders.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "Guide to Organizing Projects"
+date:   2020-11-08 12:54:04 -0700
+categories: guide projects organization folder
+---
+Once you have an idea of what you want to do for your projects, it's important to organize your working directory. If your project is organized well and everything is documented, anyone who's unfamiliar with your project should be able to look at your files and understand what you did and why. Organize everything the way that if you come back to that project in couple months or years you will able to understand what you did and why.
+
+## Directory Layout
+There are couple of reasons to have a certain structure for your project:
+1. Achieve reproducibility with minimal effort.  
+2. Optimize the time looking for the files by organizing folders based on the input and output files.  
+3. Improve the quality of the project with detailed explanations and notes as the work progresses.
+
+Basic directory layout may look like this:
+* data\
+  * raw\
+  * clean\
+* code\
+  * functions\
+    * 01_process-data.R
+    * 02_create-figures-using-processed-data.R
+  * archived\
+* output\
+  * data-exploration-figures\
+  * final-figures\
+  * descriptive-title-for-output-file-batch\
```

Comment:
Lines 16-27 should end in forward slashes `/` rather than a back slash `\`.

Reply:
Windows uses backslashes for paths, while everything else seems to use forward slashes. I guess you have a Mac computer while I have a Windows computer. I'll go ahead and change the slashes.